interface/patient_file/summary/stats.php

```diff
@@ -208,17 +208,17 @@ function load_location(location) {
 
 // Render Cards for 2 specific forms
 foreach (['treatment_protocols', 'injury_log'] as $formname) {
-    if (sqlNumRows(sqlStatement("SHOW TABLES LIKE ?", ["form_{$formname}"])) > 0) {
+    if (sqlNumRows(sqlStatement("SHOW TABLES LIKE form_{$formname}")) > 0) {
```

Comment:
fyi, can escape table name above since it's being used in a comparison (no need to change it though, since it is hard-coded 1 line above; just don't want to lead you astray in future work).